### ISSUE TITLE
bug in custom transition group

### ISSUE THREAD

**Issue by LexRiver** [kurdin/velocity-inferno#2]
@kurdin I've checked your demo, nice start!

Seems like the "Custom transition group" works incorrectly.
Animation suppose to happen on first element, not the last one, like here http://twitter-fabric.github.io/velocity-react/
I've also trying to remove element from middle and animation happens on the last element again.

Also no npm package yet?

**Comment by kurdin** [comment]
@LexRiver yes, it worked incorrect because of the bug that I fixed in inferno-transition-group. I was waiting before guys from inferno push new fixed version to npm to release this.

Now everything works as expected. Do git pull, npm i in your velocity-inferno to update inferno-transition-group. 

Now I have jquery / velocity js error on element remove/add. Working on it.

**Comment by LexRiver** [comment]
Oh yes, works really great now!

**Comment by kurdin** [comment]
@LexRiver 
I converted another React animation library to Inferno, if you need something more simple, try this: https://github.com/kurdin/inferno-animate.css

**Comment by LexRiver** [comment]
@kurdin 
I've checked that, looks really great. Nice job!
I think your next step will be some animated router -) like this https://github.com/trungdq88/react-router-page-transition
